iosViewBuilder/iosViewBuilder/ContentView.swift

```diff
@@ -6,38 +6,38 @@ import SwiftUI
 import AssessmentModelUI
 
 struct ContentView: View {
-    @StateObject var viewModel: ViewModel = .init()
+    @State var current: AssessmentState?
```

Comment:
iOS 16 changed things (sigh) and this was firing on a background thread. Updated to change on main thread.